swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = 0, category = "Clothing"):
+        self.category = category
+        self.condition = condition
```

Comment:
Per our conversation, we can use inheritance here by invoking super()

```suggestion
    def __init__(self, condition = 0):
        super().__init__(condition=condition, category="Clothing")
```